content/master/concepts/server-side-apply.md

```diff
@@ -0,0 +1,105 @@
+resources. Using server-side apply makes syncing more predictable and less
+buggy.
+
+{{<hint "tip">}}
+Server-side apply is a Kubernetes feature. Read more about server-side apply in
+the [Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/server-side-apply/).
+{{</hint>}}
+
+## Use server-side apply to sync claims with composite resources
+
+When you create a claim, Crossplane creates a corresponding composite resource.
+Crossplane keeps the claim in sync with the composite resource. When you change
+the claim, Crossplane reflects those changes on the composite resource.
+
+Read the [claims documentation]({{<ref "./claims">}}) to learn more about claims
+and how they relate to composite resources.
+
+Crossplane can use server-side apply to keep the claim in sync with the
+composite resource.
+
+Use the `--enable-ssa-claims` feature flag to enable using server-side apply.
+Read the [Install Crossplane documentation]({{<ref "../software/install#feature-flags">}})
+to learn about feature flags.
+
+If you see fields reappearing after you delete them from a claim's `spec`,
+enable server-side apply to fix the problem. Enabling server-side apply also
+fixes the problem where Crossplane doesn't delete labels and annotations from
+the composite resource when you delete them from the claim.
+
+{{<hint "important">}}
+When you enable server-side apply, Crossplane is stricter about how it syncs
+claims with composed resources:
+
+- The claim's `metadata` syncs to the composite resource's `metadata`.
+- The claim's `spec` syncs to the composite resource's `spec`.
+- The composite resource's `status` syncs to the claim's `status`.
+
+When you enable server-side apply Crossplane doesn't sync the composite resource's `metadata`
+and `spec` back to the claim's `metadata` and `spec`. It also doesn't sync the
+claim's `status` to the composite resource's `status`.
+{{</hint>}}
+
+## Use server-side apply to sync claims end-to-end
+
+To get the full benefit of server-side apply, use the `--enable-ssa-claims`
+feature flag together with composition functions.
+
+When you use composition functions, Crossplane uses server side apply to sync
+composite resources with composed resources. Read more about this in the
+[composition functions documentation]({{<ref "./composition-functions#how-composition-functions-work">}}).
+
+```mermaid
+graph LR
+  A(Claim) -- claim server-side apply --> B(Composite Resource)
+  B -- function server-side apply --> C(Composed Resource)
+  B -- function server-side apply --> D(Composed Resource)
+  B -- function server-side apply --> E(Composed Resource)
+```
+
+When you use server-side apply end-to-end there is a clear, predictable
+propagation of fields from claim to composed resources, and back:
+
+* `metadata` and `spec` flow forwards, from claim to XR to composed resources.
+* `status` flows backwards, from composed resources to XR to claim.
+
+{{<hint "important">}}
```

Comment:
It would not hurt to repeat this hint in `composition-functions.md` as well?

Reply:
The wording is different, but this constraint is documented at https://docs.crossplane.io/v1.15/concepts/composition-functions/#desired-state.